src/pages/MainPage.js

```diff
@@ -1,11 +1,19 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Paper from '@mui/material/Paper';
 import Box from '@mui/material/Box';
 import './mainpage.css'
 import { DrawerBar } from '../components/DrawerBar'
+import { DataGridCrypto } from '../components/DataGrid/DataGridCrypto'
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
 
 const MainPage = () => {
-    
+    const [value, setValue] = useState('one');
+    const [buy, setBuy] = useState(true)
+
+    const handleChange = (event, newValue) => {
```

Comment:
where do you use event parameter?

Reply:
I am actually not using it, but according to the documentation, the Tabs' onChange props requires a function with event and a value as arguments. I tried to remove the event parameter, but onChange didn't working properly without it